app/models/case_contact.rb

```diff
@@ -114,6 +114,10 @@ class CaseContact < ApplicationRecord
     end
   }
 
+  scope :with_casa_case, ->(case_ids) {
+    where(casa_case_id: case_ids) if case_ids.present?
```

Comment:
Tell me more about the conditional here

Reply:
`case_ids` can be nil or empty, leading to a blank report since there are no reports with null `casa_case_id`. So to keep the previous behavior when `casa_case_id` is not provided, I need to ensure its validation, just like in the other scopes.

Reply:
thanks!